nep-17.mediawiki

```diff
@@ -1,7 +1,7 @@
 <pre>
   NEP: 17
   Title: Token Standard
-  Author: Erik Zhang <erik@neo.org>
+  Author: Erik Zhang <erik@neo.org>, Fernando Díaz Toledano <shargon@gmail.com>, Igor Machado Coelho <igor.machado@gmail.com>, Jaime Kindelan <jaime@red4sec.com>, Roman Khimov <roman@nspcc.ru>, Vitor Nazário Coelho <vncoelho@gmail.com>
```

Comment:
```suggestion
  Author: Erik Zhang <erik@neo.org>, Igor Machado Coelho <igor.machado@gmail.com>, Jaime Kindelan <jaime@red4sec.com>, Roman Khimov <roman@nspcc.ru>, Vitor Nazário Coelho <vncoelho@gmail.com>
```
I doesn't feel like an author

Reply:
Ok,Shargon, I believe so, you are contributing to make this standard solid.

Reply:
If you are not going to be include yourself I will remove mine because the list will become quite strict.

We also need to check with previous nep5 authors.